tasks/list-active-rev.js

```diff
@@ -0,0 +1,28 @@
+const gulp = require('gulp');
+const gutil = require('gulp-util');
+
+const { getRedisClient, getConfigFor } = require('./utils');
+const { some } = require('bluebird');
+
+async function getActiveRevision(client, config) {
+  const activeRev = await client.get(config.mainRevKey);
+
+  gutil.log(`${config.mainRevKey}=${activeRev}`);
+}
+
+async function printActiveRevision(config) {
+  const client = await getRedisClient(config);
+
+  await Promise.all(
+    (config.files || [config]).map((fileConfig) =>
+      getActiveRevision(client, Object.assign({}, config, fileConfig)),
```

Comment:
*question & suggestion*: So in this implementation you are expecting that print inside of `getActiveRevision` function? If so I don't like that design. `getActiveRevision` should rather return the revision.. and then we can do something you would like in the `printActiveRevision`

Reply:
Thank you for pointing that out. I changed `getActiveRevision` to only return the value